src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -40,13 +40,15 @@ public void tearDown() {
   public void testSaveAndLoadUsers() throws PersistentDataStoreException {
     UUID idOne = UUID.randomUUID();
     String nameOne = "test_username_one";
+    String passwordOne = "password_one";
     Instant creationOne = Instant.ofEpochMilli(1000);
-    User inputUserOne = new User(idOne, nameOne, creationOne);
+    User inputUserOne = new User(idOne, nameOne, passwordOne, creationOne);//made change
```

Comment:
you don't need to add the "//made change" comment - in the git commit history, we'll be able to look back and see that this change was associated with this commit :)

(here and other places)

Reply:
Done